src/NUnitFramework/tests/SynchronizationContextTests.cs

```diff
@@ -104,7 +104,7 @@ public static void AsyncDelegatesAreExecutedOnStaThread(AsyncExecutionApiAdapter
             });
         }
 
-#if !NETCOREAPP2_1
+#if UseWindowsFormsAndWPF || NETFRAMEWORK
```

Comment:
This implies to me that UseWindowsFormsAndWPF isn't necessarily set on .NET Framework. What do you think about removing the `|| NETFRAMEWORK` part?

Reply:
I guess i got confused by the use of mono in the unittests.
Will reduce to one define being used.